src/Components/Buttons/Popover/Popover.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+export class Popover extends Component {
```

Comment:
I think better to create HOC instead of component
we can use HOC with any component and show popovers anywhere

Reply:
https://reactjs.org/docs/higher-order-components.html